tests/unit/statistical_testing/QuadraticTimeMMD_unittest.cc

```diff
@@ -354,7 +354,7 @@ TEST(QuadraticTimeMMD, perform_test_permutation_biased_full)
 	// assert against local machine computed result
 	mmd->set_statistic_type(ST_BIASED_FULL);
 	float64_t p_value=mmd->compute_p_value(mmd->compute_statistic());
-	EXPECT_NEAR(p_value, 0.0, 1E-10);
+	EXPECT_NEAR(p_value, 0.8, 1E-10);
```

Comment:
hi, @karlnapf @vigsterkr. yeah, this test is pretty meaningless... it's just there to ensure that compute_p_value isn't broken and gives the same result every time when running with same seed. so whatever p-value comes here, maybe we shouldn't read too much into it. It's just 10 null samples.